pkg/k8sutil/stunnel.go

```diff
@@ -2,7 +2,8 @@ package k8sutil
 
 import (
 	"fmt"
-	"k8s.io/api/core/v1"
+	"os"
+        "k8s.io/api/core/v1"
```

Comment:
remove extra space

Reply:
Let me clean up, it is something to do with my VS code formatting, I will remove it, this is annoying.